doc/classes/UndoRedo.xml

```diff
@@ -141,6 +141,20 @@
 				This is useful mostly to check if something changed from a saved version.
 			</description>
 		</method>
+		<method name="has_redo">
+			<return type="bool">
+			</return>
+			<description>
+				returns true if redo available.
+			</description>
+		</method>
+		<method name="has_undo">
+			<return type="bool">
+			</return>
+			<description>
+				returns true if undo available.
```

Comment:
Same thing about capital letter and `[code]true[/code]`.

I'd suggest "Returns [code]true[/code] if an 'undo' action is available."